public/javascripts/script.js

```diff
@@ -1,31 +1,71 @@
-// This will print in the wrong order
-// we added it for you to test to make sure data is loaded
-// 🚨🚨🚨 comment out the next 3 lines when you start working on the code
-for (let i = 0; i < mashPotatoes.length; i++) {
-  //addFood(steak[i], '#steak');
-  console.log(mashPotatoes[i])
-}
-
 // Iteration 1 using callbacks
-addFood(steak[0], '#steak', () => {
-  // ... your code here
-  addFood(steak[1], '#steak', () => {
-
-  })
-});
 
+let steakImg = document.createElement("img");
+function prepareSteak() {
+  function showSteak() {
+    let table = document.getElementById("table");
+    steakImg.setAttribute("src", "../public/images/steak.jpg");
+    table.appendChild(steakImg);
+  }
 
+  addFood(steak[0], "#steak", () => {
+    addFood(steak[1], "#steak", () => {
+      addFood(steak[2], "#steak", () => {
+        addFood(steak[3], "#steak", () => {
+          addFood(steak[4], "#steak", () => {
+            addFood(steak[5], "#steak", () => {
+              addFood(steak[6], "#steak", () => {
+                addFood(steak[7], "#steak", () => {
+                  showSteak();
+                });
+              });
+            });
+          });
+        });
+      });
+    });
+  });
+}
 
 // Iteration 2 using `.then()`
-addFood(mashPotatoes[0], '#mashPotatoes').then(() => {
-  // ... your code here
-  addFood(mashPotatoes[1], '#mashPotatoes')
-});
+function preparePotate() {
+  function showPotate() {
+    let potateImg = document.createElement("img");
+    potateImg.setAttribute("src", "../public/images/mashPotatoes.jpg");
+    table.appendChild(potateImg);
+  }
+
+  addFood(mashPotatoes[0], "#mashPotatoes").then(() => {
```

Comment:
In order to make the code looks cleaner you can use this structure, link: https://prnt.sc/u4u573

Reply:
Same in the following lines